RecoJets/JetProducers/plugins/SubEventGenJetProducer.h

```diff
@@ -10,6 +10,7 @@
 
  ************************************************************/
 
+#include <memory>
```

Comment:
As such `<memory>` is included twice, in the header and in the implementation: please remove from the implementation file